internal/pkg/aws/cloudformation/cloudformation.go

```diff
@@ -227,6 +228,11 @@ func (c *CloudFormation) events(stackName string, match eventMatcher) ([]StackEv
 	return events, nil
 }
 
+// ListStacksWithPrefix returns all the stacks in the current AWS account.
+func (c *CloudFormation) ListStacksWithPrefix(prefix string) ([]StackSummary, error) {
```

Comment:
I wonder if we could make this method more versatile:
```suggestion
func WithPrefix(prefix string) ListStackOpts {
    return func(in *cloudformation.DescribeStacksInput) {
        in.StackName = prefix
    }
}

func (c *CloudFormation) ListStacks(opts... ListStackOpts) ([]StackDescription, error) {
```

Reply:
For now since we're not using the status filter for anything I'll leave this as is, but in the future we can define a custom listStackInput datatype that handles these options so we can make nice calls like 
```go
ListStacks(WithPrefix("task-"), WithFilter(SuccessfulStatuses))
```